recipes/containers-common/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.26.3" %}
+
+package:
+  name: containers-common
+  version: {{ version }}
+
+source:
+  - url: https://github.com/containers/common/archive/v{{ version }}.tar.gz
+    sha256: c0b62b6cc7be3304771334fa765fc7d6bf9f9cb65ec44f9776a3e5843e0f6da5
+    folder: common
+  - url: https://github.com/containers/skopeo/archive/v1.2.0.tar.gz
+    sha256: 113290f6747b7a9946ddf50ad1a7d924d3e507fe923b2e0460df1e6823de0ffb
+    folder: skopeo
+  - url: https://github.com/containers/buildah/archive/v1.17.0.tar.gz
```

Comment:
Is there anything that can be done to help ensure these stay in sync?

Reply:
Not yet. I believe some things from `skopeo` will be added to `containers-common`. The move to a shared configuration source is comparatively new, hence still in change.

That said, I should add a comment in the recipe!